MoreLinq.Test/TagFirstLastTest.cs

```diff
@@ -22,12 +22,29 @@ namespace MoreLinq.Test
     [TestFixture]
     public class TagFirstLastTest
     {
+        [Test]
+        public void TagFirstLastEvaluatesSourceLazily()
+        {
+            var source = MoreEnumerable.From(() => 123, () => 456, () => throw new TestException());
```

Comment:
Use `BreakingFunc` instead of throwing `TestException`:

```suggestion
            var source = MoreEnumerable.From(() => 123, () => 456, BreakingFunc.Of<int>());
```